package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@streammedev/hermes",
-  "version": "1.4.4",
+  "version": "1.4.5-BETA",
```

Comment:
Preferably don't bump the version in a PR branch.  We can do the version bump with `npm version minor` straight to `1.4.5` after we merge up these PRs.

Reply:
Aye! Will do. Version to 2.0.0 on master after merge per our conversation